packages/@office-iss/react-native-win32/src/Libraries/Components/Button.win32.js

```diff
@@ -328,6 +328,8 @@ class Button extends React.Component<ButtonProps> {
       disabled: ariaDisabled ?? accessibilityState?.disabled,
       expanded: ariaExpanded ?? accessibilityState?.expanded,
       selected: ariaSelected ?? accessibilityState?.selected,
+      multiselectable: accessibilityState?.multiselectable,
```

Comment:
Do we also need `// [Windows` or `// Win32` comments here? I don't actually know, I'm not sure what the style guide for RNW is

Reply:
I'm not sure either but I suppose it couldn't hurt to add. I'll try to follow the style as best I can.